Add docs on `run_from` fields (#33466)
elastic/beats: heartbeat/docs/heartbeat-observer-options.asciidoc

[[configuration-observer-options]]
== Add observer and geo metadata

Use the `heartbeat.run_from` option to set the geographic location fields relevant to a given heartbeat instance.

The `run_from` option is used to label the geographic location where the monitor is running.
Note, you can also set the `run_from` option on an individual monitor to apply a unique setting to just that monitor.

The `run_from` option takes two top-level fields:

* `id`: A string used to uniquely identify the geographic location. It is indexed as the `observer.name` field.

> * `geo`: A map conforming to https://www.elastic.co/guide/en/ecs/current/ecs-geo.html[ECS geo fields]. It is indexed under `observer.geo`.

Reviewer: ```suggestion
* `geo`: A map conforming to {ecs-ref}/ecs-geo.html[ECS geo fields]. It is indexed under `observer.geo`.
```

[[configuration-observer-options]]
== Add observer and geo metadata

Use the `heartbeat.run_from` option to set the geographic location fields relevant to a given heartbeat instance.

> Determines which geographic location to label this monitor as running from. Note, this
> can also be handled by setting the `run_from` option on an individual monitor to apply
> a unique setting to just that monitor.

Reviewer: A couple thoughts:

* The first sentence here is unclear. What is doing the determining?
* The second sentence could be simplified.
* Is something like the suggestion below still accurate? Feel free to adjust as needed.

```suggestion
The `run_from` option is used to label the geographic location where the monitor is running.
Note, you can also set the `run_from` option on an individual monitor to apply a unique setting to just that monitor.
```

> Takes two top-level fields: `id` and `geo`, where `id` is a string used to uniquely identify
> the geographic location, and indexed as the `observer.name` field, and `geo` which is a map
> conforming to https://www.elastic.co/guide/en/ecs/current/ecs-geo.html[ECS geo fields] 
> and is indexed under `observer.geo`.

Reviewer: An unordered list might be easier to read:

```suggestion
The `run_from` option takes two top-level fields:

* `id`: A string used to uniquely identify the geographic location. It is indexed as the `observer.name` field.
* `geo`: A map conforming to https://www.elastic.co/guide/en/ecs/current/ecs-geo.html[ECS geo fields]. It is indexed under `observer.geo`.
```
